rotkehlchen/chain/ethereum/trades.py

```diff
@@ -201,19 +201,19 @@ def to_db_tuple(self) -> AMMSwapDBTuple:
             is_token0_unknown,
             str(self.token0.ethereum_address),
             str(self.token0.symbol),
-            str(self.token0.name),
+            self.token0.name,
```

Comment:
is this always guaranteed to be a string? I guess yes, but I have to ask.

Reply:
I have to allow `None` because in Balancer now we may have `UnknownEthereumToken` without `name` and `decimal` (requesting the swaps there is a pool where its controller has removed the tokens). This is why these fields are now `Optional` in `AMMSwapDBTuple`.